esmvalcore/_main.py

```diff
@@ -266,7 +268,8 @@ def show(recipe):
         recipe: str
             Name of the recipe to get, including any subdirectories.
         """
-        from ._config import DIAGNOSTICS, configure_logging
+        from .config._diagnostics import DIAGNOSTICS
+        from .config._logging import configure_logging
```

Comment:
Because it gives a very slow user experience, e.g. if you run `esmvaltool --help`, you expect a prompt response. However, if it needs to import everything and load all the cmor tables, this takes up to a few seconds and gives the command line a very sluggish feel.

Reply:
ah gotcha, very cool point I didn't think of :medal_military: